App.js

```diff
@@ -1,68 +1,60 @@
 import React, { Component } from 'react';
-import {
-  AppRegistry,
-  AsyncStorage,
-  Image,
-  StyleSheet,
-  View,
-  LayoutAnimation
-} from 'react-native';
-
-import { colors } from 'config';
-
-//Routes
-import { GitPoint } from './routes';
-
-// Redux Store
-import { configureStore } from './root.store';
 import { Provider } from 'react-redux';
-
+import { AppRegistry, AsyncStorage, Image, StyleSheet, View, LayoutAnimation } from 'react-native';
 import { persistStore } from 'redux-persist';
 import createEncryptor from 'redux-persist-transform-encrypt';
-
-// Device Info
 import DeviceInfo from 'react-native-device-info';
-
-// md5
 import md5 from 'md5';
 
+import { colors } from './src/config';
+import { GitPoint } from './routes';
+import { configureStore } from './root.store';
+
+const styles = StyleSheet.create({
+  logoContainer: {
+    backgroundColor: colors.white,
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  logo: {
+    width: 100,
+    height: 100,
+  },
+});
+
 class App extends Component {
   constructor() {
     super();
 
     this.state = {
-      rehydrated: false
+      rehydrated: false,
     };
   }
 
   componentWillMount() {
     const encryptor = createEncryptor({
-      secretKey: md5(DeviceInfo.getUniqueID())
+      secretKey: md5(DeviceInfo.getUniqueID()),
     });
 
-    persistStore(
-      configureStore,
-      { storage: AsyncStorage, transforms: [encryptor] },
-      () => {
-        this.setState({ rehydrated: true });
-      }
-    );
+    persistStore(configureStore, { storage: AsyncStorage, transforms: [encryptor] }, () => {
+      this.setState({ rehydrated: true });
+    });
   }
 
   componentWillUpdate() {
     LayoutAnimation.spring();
   }
 
   render() {
-    if (!this.state.rehydrated)
+    if (!this.state.rehydrated) {
       return (
         <View style={styles.logoContainer}>
-          <Image
-            style={styles.logo}
-            source={require('./src/assets/logo-black.png')}
-          />
+          <Image style={styles.logo} source={require('./src/assets/logo-black.png')} />
```

Comment:
I'd like to enforce 
```js
<Image
    style={styles.logo}
    source={require(....)}
/>
```
instead of a _one liner_ do you think we can do that with prettier?